src/web/index.html

```diff
@@ -2,10 +2,12 @@
 <html lang="en">
   <head>
     <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
-    <title>code.pyret.org</title>
+    <title>Student Dashboard - code.pyret.org</title>
```

Comment:
I believe the teacher one says "Teacher Dashboard." Another consideration is how do we tell teachers that they can access a separate dashboard if the "landing page" is the student version.